src/openrct2/object/SceneryObject.h

```diff
@@ -25,15 +25,14 @@ class SceneryObject : public Object
     }
     virtual ~SceneryObject() = default;
 
-    const rct_object_entry* GetPrimarySceneryGroup()
+    const ObjectEntryDescriptor& GetPrimarySceneryGroup()
```

Comment:
```suggestion
    const ObjectEntryDescriptor& GetPrimarySceneryGroup() const
```

Method should probably be `const` as well